docs/reference/citing.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: Citing ONNX Runtime
+description: Instructions for citing and using ONNX Runtime logos
 parent: Reference
-nav_exclude: true
+nav_order: 9
```

Comment:
I think this should be 7? 
Also, there is currently a web app tutorial page on this list that seems a little out of place. I wonder if we should have dedicated pages on the sidebar for Mobile and Web rather than clubbing them under References.